bson.cpp

```diff
@@ -424,6 +428,91 @@ void VariantToBsonConverter::_convertSerializable(bson_t *bson, const char *key,
 }
 /* }}} */
 
+/* {{{ MongoDriver\Driver\ReadConcern */
+void VariantToBsonConverter::_convertReadConcern(bson_t *bson, const char *key, Object v)
+{
+	MongoDBDriverReadConcernData* data = Native::data<MongoDBDriverReadConcernData>(v);
+	const char *rp_string = mongoc_read_concern_get_level(data->m_read_concern);
+	bson_t child;
+
+	bson_append_document_begin(bson, key, -1, &child);
+
+	bson_append_utf8(&child, "level", -1, rp_string, -1);
```

Comment:
Someone pointed this out the other day (that they aren't documented), I've created a ticket for it: https://jira.mongodb.org/browse/CDRIVER-1531

The uppercase macros skipping the len is definitely supported features, and is mentioned a littlebit in the docs: http://mongoc.org/libbson/current/creating.html